docs/additional-deployment-options/api_gateway_logging.md

```diff
@@ -0,0 +1,92 @@
+---
+id: api-gateway-logging
+title: API Gateway Logging
+hide_title: true
+---
+
+# API Gateway Logging
+
+## Enabling API Gateway logging
+
+In order to enable API Access and Execution logging, configure the Cumulus deployment by setting `log_api_gateway_to_cloudwatch` on the `cumulus` module .
```

Comment:
```suggestion
In order to enable API Access and Execution logging, configure the Cumulus deployment by setting `log_api_gateway_to_cloudwatch` on the `cumulus` module to `true`.
```

Reply:
I suppose this is redundant given L13, my immediate reaction reading that sentence was "to what?"

Reply:
Alternately:


In order to enable API Access and Execution logging, configure the Cumulus deployment by setting `log_api_gateway_to_cloudwatch` on the `cumulus` module:

```hcl
log_api_gateway_to_cloudwatch = true
```